tests/Resources/utilities/assertions.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+/* globals OS_ANDROID,OS_IOS */
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/u...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/utilities/assertions.js;
  Line: 2;
-->

- :warning: tests/Resources/utilities/assertions.js line 2 – 'OS_IOS' is defined but never used. (no-unused-vars)